app/views/editions/_reviewer_field.html.erb

```diff
@@ -1,3 +1,3 @@
-<%= form_group(f, :reviewer) do %>
+<%= form_group(f, :reviewer, label: "Reviewer") do %>
```

Comment:
Does this conflict with our translations work? Should this refer to a i18n yaml property instead?

Reply:
The translation work is only being done on the frontend apps, so publisher isn't affected.  If we decide to translate publisher, then we'll need to audit all the hardcoded strings (like we did in the frontend apps).